README.md

```diff
@@ -1,69 +1,89 @@
+
 To use a secret in a GitHub job, you can use a GitHub action to retrieve secrets from AWS Secrets Manager and add them as masked [Environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables) in your GitHub workflow. For more information about GitHub Actions, see [Understanding GitHub Actions](https://docs.github.com/en/actions/learn-github-actions/understanding-github-actions) in the *GitHub Docs*.
-​
 
 When you add a secret to your GitHub environment, it is available to all other steps in your GitHub job. Follow the guidance in [Security hardening for GitHub Actions](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions) to help prevent secrets in your environment from being misused.
-​
-
-Environment variables have stricter naming requirements than secrets, so this action transforms secret names to meet those requirements. For example, the action transforms lowercase letters to uppercase letters. Because of the transformed names, two environment variables might end up with the same name. For example, a secret named "MySecret" and a secret named "mysecret" would both become environment variables named "MYSECRET". In this case, the action will fail, because environment variable names must be unique. Instead, you must specify the name you want to use for the environment variable.
-​
 
-You can set the entire string in the secret value as the environment variable value, or if the string is JSON, you can parse the JSON to set individual environment variables for each JSON key-value pair. If the secret value is a binary, the action converts it to a string.
-​
+You can set the entire string in the secret value as the environment variable value, or if the string is JSON, you can parse the JSON to set individual environment variables for each JSON key\-value pair. If the secret value is a binary, the action converts it to a string.
 
 To view the environment variables created from your secrets, turn on debug logging. For more information, see [Enabling debug logging](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/enabling-debug-logging) in the *GitHub Docs*.
-​
-​
-### Prerequisites
-​
-To use this action, you first need to configure AWS credentials and set the AWS Region in your GitHub environment by using the `configure-aws-credentials` step. Follow the instructions in [Configure AWS Credentials Action For GitHub Actions](https://github.com/aws-actions/configure-aws-credentials) to **Assume role directly using GitHub OIDC provider**. This allows you to use short-lived credentials and avoid storing additional access keys outside of Secrets Manager.
-​
+
+To use the environment variables created from your secrets, see [Environment variables](https://docs.github.com/en/actions/learn-github-actions/environment-variables) in the *GitHub Docs*.
+
+## Prerequisites<a name="retrieving-secrets_github_prereq"></a>
```

Comment:
The github itself already generates anchors for `#` (e.g. [#prerequisites](https://github.com/aws-actions/aws-secretsmanager-get-secrets#prerequisites)). Why do we need additional `retrieving-secrets_****`? Is the motivation having stable links used in external documentation not dependent on possible language change?

Reply:
> The github itself already generates anchors for `#` (e.g. [#prerequisites](https://github.com/aws-actions/aws-secretsmanager-get-secrets#prerequisites)). Why do we need additional `retrieving-secrets_****`? Is the motivation having stable links used in external documentation not dependent on possible language change?

removed the `<a..>` tags!